include/RAJA/policy/loop/scan.hpp

```diff
@@ -0,0 +1,158 @@
+/*!
```

Comment:
Similar to the comment on the loop/reduce.hpp file. We shouldn't need this file and instead just use the seq scan stuff.